index.js

```diff
@@ -1,15 +1,17 @@
-require('dotenv').config();
-const express = require('express');
-const votingsRoutes = require('./routes/votings/votings-api');
-const users = require('./routes/users/users-api');
-
-const app = express();
-const PORT_VAR = process.env.API_PORT
-
-// Mount route files
-app.use('/api/userVotes', votingsRoutes);
-app.use('/api/users', users);
-
-app.listen(PORT_VAR, () => {
-  console.log(`Server started on port ${PORT_VAR}`);
-});
+require('dotenv').config();

```

Comment:
A good practice is to add to version controll a sample/template env file, e.g. `sample.env` which would list all the required env vars with some dummy values (ideally values which can be used for development), because now you need to scan the whole codebase for `process.env` occurences to know which env vars are required to run the app.

You should then also add instruction to the readme to create the `.env` file from the template:

```bash
cp sample.env .env
```

Reply:
??

Reply:
I do not understand.